test/unit/js/pauseResume.test.js

```diff
@@ -10,13 +10,46 @@ QUnit.config.reorder = false;
 var hooks = {
 	beforeEach: function() {
 		this.$el = $("#box1");
+		this.$el.each(function() {
```

Comment:
The way of handling will dependent of data types. So if we're handling array is ok using array methods, but if we handle non array values is quite considerable use non array method for that.

In my opinion for that case I'd write like:
```js
this.$el = $("#box1");
delete this.$el.get(0).__aniProps;

// but, also below one is ok
delete $el[0].__aniProps;
```

But, it's not a big deal issue. It's ok what is written now.